opm/parser/eclipse/EclipseState/Aqucon.hpp

```diff
@@ -0,0 +1,59 @@
+/*
+  Copyright (C) 2020 Equinor ASA
+  Copyright (C) 2020 SINTEF Digital
+
+  This file is part of the Open Porous Media project (OPM).
+
+  OPM is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  OPM is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  ?You should have received a copy of the GNU General Public License
+  along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef OPM_AQUCON_HPP
+#define OPM_AQUCON_HPP
+
+#include <opm/parser/eclipse/EclipseState/Grid/FaceDir.hpp>
+// #include <unordered_map>
+#include <map>
+#include <array>
+
+namespace Opm {
+
+    class EclipseGrid;
+    class Deck;
+    class DeckRecord;
+
+    struct NumAquiferCon {
```

Comment:
Can this share some code with the already existing implementation in `Aquancon`?